src/components/Timer.js

```diff
@@ -0,0 +1,27 @@
+import { CHEMICAL_ELEMENTS } from '../model/constants';
+
+const Timer = () => {
+  const timerElement = document.createElement('div');
+  timerElement.classList.add('timer');
+
+  const numberSection = document.createElement('div');
+  numberSection.textContent = '60';
+  numberSection.classList.add('timer__number');
+
+  timerElement.appendChild(numberSection);
+
+  const textSection = document.createElement('div');
+  textSection.textContent = 'Nd';
+  textSection.classList.add('timer__text');
+
+  timerElement.appendChild(textSection);
+
+  timerElement.updateTime = (secondsLeftInGame) => {
+    numberSection.textContent = secondsLeftInGame;
+    textSection.textContent = CHEMICAL_ELEMENTS[secondsLeftInGame - 1];
+  };
```

Comment:
Czy to zadziała dla zera czy zero tu nigdy nie trafia?

Reply:
jak jest 0 to czas się kończy i komponent jest usuwany, więc zero nigdy tam nie trafia. a przynajmniej nie powinno :D